### DISCUSSION TITLE
Sort "all artists" by date added?

### DISCUSSION THREAD

**Discussion by alan12345678** [SoongNoonien/mpdevil#66] [Q&A]
Just installed mpdevil. Great! I'm replacing Cantata as it's no longer being developed. The one feature I've come to rely on across the years is that albums and tracks are displayed as "most recent" i.e. by date added to library.

Can I configure mpdevil to do this? Seems like such an obvious and simple thing... but that's easy to say. If I understand correctly, switching "sort by year" prefs makes no real difference?

**Comment by SoongNoonien** [comment]
> Can I configure mpdevil to do this?

Currently not as mpd doesn't offer a way to access the time a file was added to the library. The only time that is accessible is the modification time of the audio files. Either way it would be hard to sort the albums by that time as it could vary across the tracks of one album. Probably cantata has an extra database to store this time, but this is not an option for me.

> If I understand correctly, switching "sort by year" prefs makes no real difference?

The "sort by year" option simply sorts the albums by the "date" tag instead of their name.
